src/features/MainScreen/index.tsx

```diff
@@ -30,6 +30,29 @@ export const MainScreen: FC = () => {
 		});
 	}, [tagsRegistry, updateNotes]);
 
+	useEffect(() => {
+		const preventTextInsert = (e: MouseEvent) => {
+			if (e.button !== 1) return;
+
+			const isEditable = e.composedPath().some((element) => {
+				return (
+					element instanceof HTMLInputElement ||
+					element instanceof HTMLTextAreaElement ||
+					(element instanceof HTMLElement && element.isContentEditable)
+				);
+			});
+
+			if (!isEditable) {
+				// Prevent a KDE/X11 issue where pressing the middle mouse button pastes text
+				// from the clipboard into the focused element, even when clicking another element
```

Comment:
Move description at top of this function. We should explain such things at start of the code